addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -86,7 +86,8 @@ def _compute_l10n_latam_manual_document_number(self):
         remaining.l10n_latam_manual_document_number = False
 
     def _is_manual_document_number(self):
-        return self.journal_id.type == 'purchase'
+        # Manual number when document number is to be typed by the user, aka issued by suppliers
+        return self.is_purchase_document()
```

Comment:
@pepetreshere FYI this modification (and related in commit d6da280afe1d9e6a84c330f10ebf768a9130b09c) is going away